hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/CompactionUtil.java

```diff
@@ -189,11 +192,13 @@ public static void rollbackEarliestCompaction(HoodieFlinkTable<?> table, Configu
             instant.getState() == HoodieInstant.State.INFLIGHT).firstInstant();
     if (earliestInflight.isPresent()) {
       HoodieInstant instant = earliestInflight.get();
-      String currentTime = table.getMetaClient().createNewInstantTime();
```

Comment:
why cann't we just use `table.getMetaClient().createNewInstantTime(false)`.

Reply:
It is about the intention of the code. `createNewInstantTime` methods are meant to be used in the context of creating a new transaction ID. It has some more overhead than simply generating the time as a string.